dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProcessDefinitionServiceImpl.java

```diff
@@ -3054,4 +3058,48 @@ public void deleteOtherRelation(Project project, Map<String, Object> result, Pro
 
     }
 
+    /**
+     * Batch release process definitions by code states.
+     *
+     * @param loginUser    Login user.
+     * @param projectCode  Project code.
+     * @param codeStates   Code states in JSON format.
+     * @return Result of the batch release process definitions.
+     */
+    @Override
+    @Transactional
+    public Map<String, Object> batchReleaseProcessDefinitions(User loginUser, long projectCode, String codeStates) {
+        Map<String, Object> result = new HashMap<>();
+        if (StringUtils.isEmpty(codeStates)) {
+            log.error("Parameter codeStates is empty, projectCode is {}.", projectCode);
+            putMsg(result, Status.PROCESS_DEFINITION_CODES_IS_EMPTY);
+            throw new ServiceException(Status.PROCESS_DEFINITION_CODES_IS_EMPTY);
+        }
+
+        try {
+            Map<String, String> codeStateMap = new Gson().fromJson(codeStates, new TypeToken<Map<String, String>>() {
```

Comment:
We've already have `org.apache.dolphinscheduler.common.utils.JSONUtils`. Please do not introduce new json parsing method.

Reply:
Okay, I'll make the necessary modifications